app/jobs/reports/authentication_report.rb

```diff
@@ -0,0 +1,41 @@
+require 'reporting/authentication_report'
+
+module Reports
+  class AuthenticationReport < BaseReport
+    REPORT_NAME = 'authentication-report'
+
+    attr_accessor :report_date
+
+    def perform(report_date)
+      return unless IdentityConfig.store.s3_reports_enabled
+      self.report_date = report_date
+
+      report_configs.each do |report_hash|
+        issuers = report_hash['issuers']
+        emails = report_hash['emails']
+        name = report_hash['name']
+        data = report_maker(issuers).to_csv
+
+        emails.each do |email|
+          ReportMailer.authentication_report(
```

Comment:
I think this might be a good fit to use `ReportMailer.tables_report` potentially? It will pretty render the data as tabular data as well as attach the CSV (see the recent "Monthly Key Metrics Report" email you likely received this weekend?)

Reply:
oooh yes!!!! i love it! thanks!

Reply:
updated in https://github.com/18F/identity-idp/pull/9295/commits/ff87ad94eaccbe4164feca2b665ffca7029ae099